### DISCUSSION TITLE
verdaccio v5.13.1 load just on name or ip

### DISCUSSION THREAD

**Discussion by monrah1990** [verdaccio/verdaccio#3254] [Q&A]
Sorry, my English is not so good. I'm trying to describe my problem. I hope you can help me. Thank you!

we used verdaccio v4.10.0 and decided to update verdaccio.
On another server I install verdaccio v5.13.1 and copy storage from old server to new server.

(assume IP of server is 192.168.12.11 and  for this IP, "npm.test.ir" is defined in the DNS server or in the /etc/hosts)
Now I have a problem
I run verdaccio (pm2) . when I write the  IP of server in the browser, the storage loads correctly. Then in the new tab enter "npm.test.ir " and I get "No Package Published Yet."
log:  200, user: null(ip of user via 127.0.0.1), req: 'GET /', bytes: 0/526

Also if I restart verdaccio and first in the browser enter  "npm.test.ir " storage loads correctly and everything is OK , and then if I enter the IP in the new tab I get "No Package Published Yet."

It depends which request is received to verdaccio first.

**Comment by juanpicado** [comment]
Why would you like to use a service in two domains? I'd recommend use one domain. Is that an issue for you?

**Reply by monrah1990** [reply]
NO I don't want to use two domain.
I just want to reach to verdaccio by IP and server name parallel.
Technically when defined name for ip in DNS or /etc/host, you can reach them from both option(ip and name). right?

**Comment by monrah1990** [comment]
overall that's Ok
I think maybe it was a bug in this version and somebody have this problem.

thanks for Verdaccio

**Reply by juanpicado** [reply]
did you solve it?

**Reply by monrah1990** [reply]
unfortunately no

**Comment by monrah1990** [comment]
for testing  I installed verdaccio5.13.1 on my local machine, and added "127.0.0.1    test.ir" to /etc/hosts. then I ran verdaccio (without changing the config file ) the issue remained.
I uninstalled verdaccio and installed verdaccio 4.10.0 and tested again, It was possible to access the web UI with all packages from both (localhost:4873 and test.ir:4873)
